nucypher/cli/commands/alice.py

```diff
@@ -373,7 +398,7 @@ def grant(general_config,
     }
 
     if not ALICE.federated_only:
-        grant_request.update({'value': value})
+        grant_request.update({'value': value, 'rate': rate})
```

Comment:
@cygnusv within `grant`, do we want either `value` or `rate` to be provided but not both? i.e. perform an xor check of those cli options.

Reply:
This. You can't really provide both. You need either or.

Reply:
That's already checked here, in `BlockchainPolicy`:
https://github.com/nucypher/nucypher/blob/345a09e2f83d2f6c288bb4171337571b16f326f8/nucypher/policy/policies.py#L507